### PR TITLE
New Box Values, Delete KnownIdentifier, Delete Legacy Models

### PR DESCRIPTION
**Pull request by SeanCheatham**
## Purpose
- Incorporates changes from [this](https://github.com/Topl/protobuf-specs/pull/38) PR (remove KnownIdentifier)
- Update Box Values to
  - LVL
  - TOPL
  - Asset
  - Registration
- Delete "legacy" message types (models/ directory)
## Approach
- Merge PR#38
- Change Box Value types
- Remove "models" directory, and update lingering references
## Testing
- Used on separate development branches for quivr4s, bramblSc, and Bifrost
## Tickets
- BN-837
- TSDK-333

### CODE REVIEW

File: proto/brambl/models/box/value.proto [hunk 1]
```diff
@@ -7,24 +7,38 @@ import 'scalapb/scalapb.proto';
...
-
-    message Token {
+    message LVL {
+        quivr.models.Int128 quantity = 1 [(validate.rules).message.required = true];
+    }
+    message TOPL {
         quivr.models.Int128 quantity = 1 [(validate.rules).message.required = true];
+        // An optional staking address.
+        // An empty bytestring is interpreted as "None"
+        bytes stakingAddress = 2;
```
Comment (nandotorterolo) [inline proto/brambl/models/box/value.proto:27]: maybe we could use  [BytesValue](https://protobuf.dev/reference/protobuf/google.protobuf/#bytes-value) Well-Known Types

**Review by nandotorterolo — APPROVED** [review]

**Review by DiademShoukralla — APPROVED** [review]
LGTM! Just had a minor question

File: proto/brambl/models/box/value.proto [hunk 2]
```diff
@@ -7,24 +7,38 @@ import 'scalapb/scalapb.proto';
...
     }
     message Asset {
```
[the remark this answers is not in the document]
Comment (SeanCheatham) [reply proto/brambl/models/box/value.proto:32]: We'll most likely need to rename it to `AssetV1` (or something similar)

**Review by scasplte2 — APPROVED** [review]
👍

**Comment by scasplte2** [comment]
For additional information about the approach to refactor Known Ids, please refer to #38 as there is a diagram showing how these models relate to one another

**Pull request merged**